account_invoice_label_certisys/__openerp__.py

```diff
@@ -19,15 +19,17 @@
 ##############################################################################
 {
     'name': 'Account Invoice label certisys',
-    'description': 'Add certisys label on invoices',
+    'description': 'Add Certisys label on invoices, delivery slips and picking opperations (stock picking), and sale orders. This module should get a more generic name in the future.',
     'category': 'Invoice',
     'version': '9.0.1.0',
     'author': 'Coop IT Easy SCRLfs',
     'depends': [
         'account',
```

Comment:
you should add the dependency on stock and sales modules because trying to install it on a fresh new db may not work without the missing dependencies.